### PR TITLE
Show the user a completed screen for conviction checks required 

### PR DESCRIPTION
**Pull request by cintamani**
From: https://eaflood.atlassian.net/browse/RUBY-917

Add the code necessary to show the user a proper completion screen for when they complete their UT paying by bank transfer but they have pending conviction checks.

Depends on: https://github.com/DEFRA/waste-carriers-engine/pull/743

Screenshot 2020-03-20 at 12 23 28

### CODE REVIEW

**Comment by cintamani** [comment]
Sonar and Hakiri are won't fix / ignore

**Comment by cintamani** [comment]
Oh yes! I have to change the route, we shouldn't tell that to the user. 
About having one screen rather than 2 for application received, I don't think is a good idea because will make the view logic more complex and maintainability will suffer as a result.

File: config/routes.rb [hunk 1]
```diff
@@ -30,6 +30,12 @@
               only: :new,
               path: "registration-received-pending-payment",
               path_names: { new: "" }
+
+    resources :registration_received_pending_conviction_forms,
+              only: :new,
+              path: "registration-received-pending-conviction",
```
Comment (cintamani) [inline config/routes.rb]:
```suggestion
              path: "registration-received",
```

**Comment by irisfaraway** [comment]
Discussed this in more detail on Slack and agree to leave these controllers separate.

File: app/views/waste_carriers_engine/registration_received_pending_conviction_forms/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  <h2 class="heading-medium"><%= t(".subheading_1") %></h2>
+  <p><%= t(".paragraph_2") %></p>
+
+  <div class="panel">
+    <p><%= t(".paragraph_3") %></p>
+  </div>
+
+  <p>
+    <%= t(".contact.paragraph_1") %>
+    <a href="mailto:<%= t(".contact.email") %>nccc-carrierbroker@environment-agency.gov.uk"><%= t(".contact.email") %></a>
```
Comment (irisfaraway) [inline app/views/waste_carriers_engine/registration_received_pending_conviction_forms/new.html.erb]:
```suggestion
    <a href="mailto:<%= t(".contact.email") %>"><%= t(".contact.email") %></a>
```

Comment (cintamani) [reply app/views/waste_carriers_engine/registration_received_pending_conviction_forms/new.html.erb]: Copy-paste hell! :joy:

**Pull request merged**